discovery-provider/integration_tests/notifications/test_supporter_rank_ups.py

```diff
@@ -51,26 +50,28 @@ def test_supporter_rank_up_notification(app):
         assert len(supporter_notifications) == 3
         assert len(supporting_notifications) == 3
 
-        assert supporter_notifications[0].specifier == "1"
+        assert supporter_notifications[0].specifier == "3"
         assert supporter_notifications[0].group_id == "supporter_rank_up:3:slot:2"
         assert supporter_notifications[0].type == "supporter_rank_up"
+        assert supporter_notifications[0].type_v2 == "supporter_rank_up"
         assert supporter_notifications[0].slot == 2
         assert supporter_notifications[0].blocknumber == None
         assert supporter_notifications[0].data == {
             "rank": 3,
             "sender_user_id": 1,
             "receiver_user_id": 3,
         }
-        assert supporter_notifications[0].user_ids == [1]
+        assert supporter_notifications[0].user_ids == [3]
 
-        assert supporting_notifications[0].specifier == "3"
+        assert supporting_notifications[0].specifier == "1"
         assert supporting_notifications[0].group_id == "supporting_rank_up:3:slot:2"
         assert supporting_notifications[0].type == "supporting_rank_up"
+        assert supporting_notifications[0].type_v2 == "supporting_rank_up"
```

Comment:
just so I fully understanding, the sender sees "supporting rank up"  and the receiver sees "supporter rank up" 

maybe these things make more sense as "sender_supporter_rank_up" and "receiver_supporter_rank_up" or something like that. anyway, that'd be a big change now, but a comment at least explaining this relationship would be helpful!

Reply:
oo yea that can def be something we do down the line and that should be a very easy migration! will just have to update all the hardcoded references to it everywhere